spec/services/mailer/transactional_email_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+require 'shared/stripe'
+
+describe TransactionalEmail do
+  describe '#send' do
+    let(:brand)         { identity.brand }
+    let!(:identity)      { create(:identity) }
+    let!(:subscription) { create(:subscription, brand: brand)}
+
+    before do
+      allow(Mandrill::API).to receive(:new).and_call_original
+      # allow(Mandrill::API).to receive(:new).and_return(Mandrill::API.new('REAL_API_KEY'))
+    end
+
+    context 'welcome email' do
+      it 'sends welcome email template' do
+        VCR.use_cassette 'mandrill_response' do
+          response = described_class.welcome(brand).send
+          #expect(response.rejected).to be(nil)
```

Comment:
Yoo Rick,

So I using the `real_api_key` first time, I was to record the success response. Then check when calls are made, they receive the same.  It seems like all http calls are getting returned `disabled` and `yaml` file not being created in fixtures.